mpqp/core/circuit.py

```diff
@@ -913,14 +918,14 @@ def to_other_language(
                 cargs = []
 
                 if isinstance(instruction, CustomGate):
+                    # We reverse the target qubits to pass the unitary in the usual qubit ordering
+                    reversed_target = instruction.targets
+                    reversed_target.reverse()
                     new_circ.unitary(
                         instruction.to_other_language(),
-                        instruction.targets,
+                        reversed_target,
```

Comment:
```py
reversed(instruction.targets)
```

Reply:
Indeed, we can do that. However, `reversed(...)` returns an iterator, is `list(reversed(instructions.targets))` a suitable solution?

Reply:
Done